pkgs/applications/finance/irpf/default.nix

```
@@ -0,0 +1,59 @@
+{ lib, stdenvNoCC, jdk11, makeWrapper, hicolor-icon-theme, fetchzip, fetchgit }:
+
+let
+  year = "2022";
+  pkgbuild = fetchgit rec {
+    url = "https://aur.archlinux.org/irpf.git/";
+    rev = "dc76a6366de72e236c91986a4c2a467d2472749f";
+    sha256 = "1hql5ljwq8g9h0kbgpfyss7mcmp58pzvjs5dydxcs39lxcnf972c";
+  };
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "irpf";
+  version = "2.22.1.0";
+
+  src = fetchzip rec {
+    url = "https://downloadirpf.receita.fazenda.gov.br/irpf/${year}/irpf/arquivos/IRPF${year}-1.0.zip";
+    sha256 = "0h8f51ilvg7m6hlx0y5mpxhac90p32ksbrffw0hxdqbilgjz1s68";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  propagatedBuildInputs = [ hicolor-icon-theme ];
+
+  installPhase = ''
+    mkdir -p "$out/share/icons/hicolor/128x128/apps"
+    mkdir -p "$out/share/applications"
+    mkdir -p "$out/share/licenses/${pname}"
+    mkdir -p "$out/share/${pname}"
+    mkdir "$out/bin"
+    BASEDIR="$out/share/${pname}"
+
+    cp -rf help "$BASEDIR/"
+    cp -rf lib "$BASEDIR/"
+    cp -rf lib-modulos "$BASEDIR/"
+
+    install -Dm755 irpf.jar "$BASEDIR/${pname}.jar"
+    install -Dm644 Leia-me.htm "$BASEDIR/"
+    install -Dm644 offline.png "$BASEDIR/"
+    install -Dm644 online.png "$BASEDIR/"
+    install -Dm644 pgd-updater.jar "$BASEDIR/"
```

```suggestion
    install -Dm644 Leia-me.htm offline.png online.png pgd-updater "$BASEDIR"
```
(btw, i dont think trailing `/`'s matter here)

---

thx!